stages/org.osbuild.grub2.iso

```diff
@@ -110,6 +116,7 @@ def main(root, options):
     vendor = options["vendor"]
     kdir = options["kernel"].get("dir", "/images/pxeboot")
     kopts = options["kernel"].get("opts")
+    timeout = options["timeout"]
```

Comment:
`timeout = options.get("timeout", 0)` according to the schema.

Reply:
fixed